src/gui/sdl_mapper.cpp

```diff
@@ -2137,7 +2144,14 @@ static struct {
 	{0, SDL_SCANCODE_UNKNOWN}
 };
 
+static void ClearAllBinds(void) {
+	for(CEvent *event : events){
+		event->ClearBinds();
+	}
```

Comment:
Should this be replaced with the following?
```cpp
for (CEventVector_it eit=events.begin();eit!=events.end();eit++) {
	*eit->ClearBinds();
}
```

Reply:
No, for-each loops are fine and more readable. A space after `for` to be closer to K&R style would be nice though ;)